spec/query/match_condition_spec.rb

```diff
@@ -33,14 +33,14 @@
       end
 
       it "converts time to iso8601 and returns range" do
-        time = Time.utc(2018, 12, 27, 20, 10).in_time_zone("Pacific Time (US & Canada)")
+        time = Time.utc(2018, 12, 27, 20, 10, 0o0, 123).in_time_zone("Pacific Time (US & Canada)")
```

Comment:
Why the use of octal here? (same question elsewhere)

Reply:
Rubocop formatted it like this after Hound barked (in another PR). I don't see any specific Invoca style preferences for this one so I just went with it.

Reply:
Or (lol) are you saying this should just be `0`? If so, d'oh and I agree.

Reply:
Ah, ok. Did  you use `00`? That's an octal number in Ruby. It's best to keep with decimal a la `0`.

Reply:
Yeah, you must have had leading 0 in there, which is pretty common for dates. But unfortunately Ruby carried across the format from C that says that is an octal constant.
So: get rid of the leading 0 and also the 0c!

Reply:
Resolved.